MeetingNMusae/MeetingNMusae/Views/BestPlayerSelectView.swift

```diff
@@ -14,7 +14,7 @@ struct BestPlayerSelectView: View {
         GridItem()
     ]
     
-    @State var remainTime: Int = 10
+    @State var remainTime: Int = 1
```

Comment:
이 부분 다시 10으로 바꿔주세요. 지금 상황은 1초 후 최고의 무새 화면에서 넘어가서 기존 10초로 바꿔주세요.
```suggestion
    @State var remainTime: Int = 10
```